tests/foreman/api/test_virt_who_config.py

```diff
@@ -32,7 +32,8 @@ def test_positive_vm_create(self):
         :steps:
             1. Create new VM in a supported hypervisor
             2. Wait until the next virt-who report comes in.
```

Comment:
Its not a step at all.. remove it or update the verification section e.g. Verify the generated virt-who report from xyz place ?